README.md

```diff
@@ -16,6 +16,12 @@ To check out any local changes you've made:
 
 A local server should start and you should be able to access a local version of the website from there.
 
+### Assets Page
+
+1. To build the Assets page, it's necessary to go into the **.\generate-assets** folder and run the command:
+   1. `generate_assets.sh`
+2. If you are on Windows, it`s recommended to use WSL or Git Bash.
+
 ## Assets generation
 
 Assets are generated using data from /generate/assets and crates.io using their [datadump](https://crates.io/data-access) trough [cratesio-dbdump-lookup](https://github.com/alyti/cratesio-dbdump-lookup).
```

Comment:
```suggestion
### Assets, Errors, and Examples pages

These pages need to be generated in a separate step by running the shell scripts in the `generate-assets`, `generate-errors`, and `generate-wasm-examples` directories. On Windows, you can use [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) or [git bash](https://gitforwindows.org/).

Assets are generated from a [crates.io data dump](https://crates.io/data-access) through [cratesio-dbdump-lookup](https://github.com/alyti/cratesio-dbdump-lookup).

Please note that when unpacked, the crates.io dump fills about 500 mb.
```

I would recommend something like this.

I think it's a bit odd that this is presented as a numbered list. The content doesn't really fit that presentation. There's really only one step.

The text below is pretty confusing too. It seems to be trying to accomplish the same thing as this PR to me, so I think we can roll that into the same section.

This also isn't specific to the assets page. The same pattern applies for errors and examples, so we might as well tell users about those too.

@IceSentry any thoughts, especially wrt the `## Assets generation` section?

Reply:
I wouldn't be opposed to just dropping those sentences.